.github/workflows/releasecandidate.yml

```diff
@@ -67,3 +67,18 @@ jobs:
           slack_webhook_url: ${{secrets.SLACK_WEBHOOK_URL}}
           channel: '#team-devops'
           name: 'Workflow Status'
+    
+  run-sdk-networking-tests:
+    needs: [staticbuild, appimage]
+    if: ${{ success() }}
+    name: Trigger Networking Tests Workflow
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Workflow
+        run: |
+          curl -X POST \
+          -H "Authorization: token ${{secrets.GITHUB_TOKEN}}" \
+          -H "Accept: application/vnd.github+json" \
+          https://api.github.com/repos/viamrobotics/sdk-integration-tests/dispatches \
+          -d '{"event_type":"new_rc_branch"}'
+          
```

Comment:
Something's up with your editor here, as there's still no terminating newline. Maybe try https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files?

Reply:
thanks for the tip! will never have this issue again 😼